pkg/telemetry/cassandra_agent/cassandra_agent_config.go

```diff
@@ -22,10 +25,95 @@ import (
 var (
 	agentConfigLocation = "/opt/management-api/configs/metric-collector.yaml"
 	defaultAgentConfig  = telemetryapi.CassandraAgentSpec{
-		Endpoint: telemetryapi.Endpoint{
+		Endpoint: &telemetryapi.Endpoint{
 			Port:    "9000",
 			Address: "127.0.0.1",
 		},
+		Relabels: []promapi.RelabelConfig{
+			{
+				SourceLabels: []string{"__origname__"},
+				Regex:        "org\\.apache\\.cassandra\\.metrics\\.Table.*",
+				TargetLabel:  "should_drop",
+				Replacement:  "true",
+			},
+			{
+				SourceLabels: []string{"__origname__"},
+				Regex:        "org\\.apache\\.cassandra\\.metrics\\.table.*",
+				TargetLabel:  "should_drop",
+				Replacement:  "true",
+			},
+			{
+				SourceLabels: []string{"__origname__"},
+				Regex:        "org\\.apache\\.cassandra\\.metrics\\.table\\.live_ss_table_count",
```

Comment:
This metric name does not exists in Cassandra.

Reply:
I am building to the original [MCAC filters](https://github.com/k8ssandra/k8ssandra-operator/blob/7f5687f9b78f6189e7094b1c72a2e1913ae43eb5/pkg/telemetry/cassandra_metrics_filters.go#L16-L29) where that metric name is filtered. Perhaps this metrics differs in name between Cassandra versions.

Reply:
No, MCAC just modified the names to be non-original. The real documentation (as mentioned here: https://github.com/k8ssandra/management-api-for-apache-cassandra/blob/master/management-api-agent-common/src/main/resources/default-metric-settings.yaml#L2) is https://cassandra.apache.org/doc/latest/cassandra/operating/metrics.html